src/utils/components/Charts/utils/utils.ts

```diff
@@ -32,5 +37,48 @@ export const tickFormat =
     return '';
   };
 
-export const getPrometheusData = (response: PrometheusResponse): PrometheusValue[] =>
-  response?.data?.result?.[0]?.values;
+export const getPrometheusData = (response: PrometheusResponse): PrometheusValue[] => {
+  return response?.data?.result?.[0]?.values;
+};
+
+export const getPrometheusDataByNic = (
+  response: PrometheusResponse,
+  nic: string,
+): PrometheusResult[] => {
+  if (!response?.data?.result) {
+    return [];
+  }
+  const singleNic = response?.data?.result?.find((res) => res.metric?.interface === nic);
+  return singleNic ? [singleNic] : response?.data?.result;
+};
+
+export const allNetworks = 'All networks';
```

Comment:
Please, use THIS_FORMAT for constants like that, and place constants into _constants.ts_ file, under utils dir (if there isn't such a file, feel free to create it). It is better to have constants in a separate file :)
Additionally, don't be afraid to use the constanst where needed. For example I can see in _NetworkCharts.tsx_ in this PR 'All networks' string instead of a constant.